google-http-client/src/test/java/com/google/api/client/http/HttpResponseTest.java

```diff
@@ -461,6 +462,21 @@ public LowLevelHttpResponse execute() throws IOException {
   }
 
   public void testGetContent_gzipEncoding_finishReading() throws IOException {
+    do_testGetContent_gzipEncoding_finishReading("gzip");
+  }
+
+  public void testGetContent_gzipEncoding_finishReadingWithUppercaseContentEncoding() throws IOException {
+    do_testGetContent_gzipEncoding_finishReading("GZIP");
+  }
+
+  public void testGetContent_gzipEncoding_finishReadingWithDifferentDefaultLocaleAndUppercaseContentEncoding() throws IOException {
+    Locale originalDefaultLocale = Locale.getDefault();
+    Locale.setDefault(Locale.JAPAN);
```

Comment:
Try Turkish instead:
```
Locale turkish = Locale.forLanguageTag("tr-TR");
```

That's the one common language where i and I do not case like they do in English.

Reply:
I switched it to Turkish, but JAPAN in the existing code would have given an exception without the locale specification in the implementation.